### PR TITLE
[15.0][MIG] sale_order_line_description

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

**Comment by legalsylvain** [comment]
Could take a look on CI ?
Thanks for porting this module

**Comment by emagdalenaC2i** [comment]
@legalsylvain "on CI"? What do you mean?

**Comment by ajaniszewska-dev** [comment]
@emagdalenaC2i  pre-commit fails

**Comment by emagdalenaC2i** [comment]
@ajaniszewska-dev Thanks, fixed

File: sale_order_line_description/tests/test_sale.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def setUp(self):
         self.partner_model = self.env["res.partner"]
         self.product_model = self.env["product.product"]
         self.user_model = self.env["res.users"].with_context(
-            {"no_reset_password": True, "mail_create_nosubscribe": True}
+            **{"no_reset_password": True, "mail_create_nosubscribe": True}
```
Comment (rousseldenis) [inline sale_order_line_description/tests/test_sale.py]: You can pass them as arguments like:
`with_context(no_reset_password=True, mail_create_nosubscribe=True)`

Comment (emagdalenaC2i) [reply sale_order_line_description/tests/test_sale.py]: @rousseldenis done

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by Rad0van — APPROVED** [review]
LGTM

**Pull request merged**